config/core/resources/trigger.yaml

```diff
@@ -125,6 +125,43 @@ spec:
                     description: 'Attributes filters events by exact match on event context attributes. Each key in the map is compared with the equivalent key in the event context. An event passes the filter if all values are equal to the specified values.  Nested context attributes are not supported as keys. Only string values are supported. '
                     type: object
                     x-kubernetes-preserve-unknown-fields: true
+              filters:
+                description: 'Filters is an array of SubscriptionsAPIFilter that evaluate to true or false. If any filter expression in the array evaluates to false, the event MUST NOT be sent to the Subscriber. If all the filter expressions in the array evaluate to true, the event MUST be attempted to be delivered. Absence of a filter or empty array implies a value of true. In the event of users specifying both Filter and Filters, then the latter will override the former. This will allow users to try out the effect of the new Filters field without compromising the existing attribute-based Filter and try it out on existing Trigger objects.'
```

Comment:
```suggestion
                description: 'Filters is an array of SubscriptionsAPIFilter that evaluate to true or false. If any filter expression in the array evaluates to false, the event will not be sent to the Subscriber. If all the filter expressions in the array evaluate to true, the event will be attempted to be delivered. Absence of a filter or empty array implies a value of true. In the event of users specifying both filter and filters, then the filters field will override the filter field.'
```

Reply:
Hey @erharjotsingh it doesn't look like this change was fully resolved, would you be able to make the rest of the changes to the text?